.changeset/warm-oranges-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': patch
+---
+
+Fix metadata `fromCache` default to `true` issue when listen to cache in multi-tabs.
```

Comment:
Suggest phrasing more like:

"Fix an issue with metadata `fromCache` defaulting  to `true` when listening to cache in multi-tabs."

Should it actually be "multiple tabs?"